source/PlayerInfo.cpp

```diff
@@ -2551,14 +2605,16 @@ void PlayerInfo::ValidateLoad()
 
 	// Validate the missions that were loaded. Active-but-invalid missions are removed from
 	// the standard mission list, effectively pausing them until necessary data is restored.
-	auto mit = stable_partition(missions.begin(), missions.end(), mem_fn(&Mission::IsValid));
+	missions.sort([](const Mission &lhs, const Mission &rhs) noexcept -> bool { return lhs.IsValid(); });
+	auto isInvalidMission = [](const Mission &m) noexcept -> bool { return !m.IsValid(); };
+	auto mit = find_if(missions.begin(), missions.end(), isInvalidMission);
```

Comment:
```suggestion
	auto mit = stable_partition(missions.begin(), missions.end(), mem_fn(&Mission::IsValid));
```
I'm pretty sure you didn't change this?